ydb/core/kqp/host/kqp_runner.cpp

```diff
@@ -192,14 +194,19 @@ class TKqpRunner : public IKqpRunner {
         YQL_ENSURE(IsIn({EKikimrQueryType::Query, EKikimrQueryType::Script}, TransformCtx->QueryCtx->Type));
         YQL_ENSURE(TMaybeNode<TKiDataQueryBlocks>(query));
 
+        TypesCtx.BlockEngineMode = NYql::EBlockEngineMode::Auto;
```

Comment:
You may want to explicitly disable dq.UseWideChannels and dq.UseWideBlockChannels, since BlockEngineMode=auto will enable WideChannels and WideBlocksChannels unless they are disabled explicitly

Reply:
We already set scalar channels only mode here https://github.com/ydb-platform/ydb/blob/97103370d3942b7301dfd3d042df52dcf0bc15b5/ydb/core/kqp/opt/kqp_opt_build_txs.cpp#L26

(similar to https://github.com/ydb-platform/ydb/blob/97103370d3942b7301dfd3d042df52dcf0bc15b5/ydb/library/yql/providers/dq/provider/exec/yql_dq_exectransformer.cpp#L287)